atom/browser/api/atom_api_content_tracing.cc

```diff
@@ -115,10 +115,25 @@ v8::Local<v8::Promise> StartTracing(
   return promise->GetHandle();
 }
 
-bool GetTraceBufferUsage(
-    const base::RepeatingCallback<void(float, size_t)>& callback) {
-  return TracingController::GetInstance()->GetTraceBufferUsage(
-      base::BindOnce(callback));
+void OnTraceBufferUsageAvailable(scoped_refptr<atom::util::Promise> promise,
+                                 float percent_full,
+                                 size_t approximate_count) {
+  mate::Dictionary dict = mate::Dictionary::CreateEmpty(promise->isolate());
+  dict.Set("percentage", percent_full);
+  dict.Set("value", approximate_count);
+
+  promise->Resolve(dict.GetHandle());
+}
+
+v8::Local<v8::Promise> GetTraceBufferUsage(v8::Isolate* isolate) {
+  scoped_refptr<atom::util::Promise> promise = new atom::util::Promise(isolate);
+  bool success = TracingController::GetInstance()->GetTraceBufferUsage(
```

Comment:
Please consider making this const as it isn't modified.
```diff
- bool success = ...
+ const bool success = ...
```

Reply:
we are not that strict with const for values in the C++ code (except for making const references to avoid copies)